### PR TITLE
Added support for edges with same source and target

### PR DESCRIPTION
**Pull request by arcadiogarcia**

### CODE REVIEW

File: test/visualData.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ module powerbi.extensibility.visual.test {
             ["Harper", "USA"],
             ["Logan", "Brazil"],
             ["Ella", "Canada"],
+            ["USA", "USA"]
```
Comment (savvinsergey) [inline test/visualData.ts]: I guess, first value must be "name". This is not so matter, but still...

Comment (uve) [reply test/visualData.ts]: @arcadiogarcia, could you please check it.
After that we'll merge your pull request.

Comment (arcadiogarcia) [reply test/visualData.ts]: The thing is that I need both strings to be the same, but I couldn't find a country name that can be used as a person name too.

Comment (uve) [reply test/visualData.ts]: Sorry I didn't get it. All lines in dataset look like this:
 ["William", "Brazil"],
 ["Olivia", "USA"],

So first parameter is person name, second is a country. Why do you need to change this template and use the following string: ["USA", "USA"]  ?

Comment (arcadiogarcia) [reply test/visualData.ts]: My change allows to render edges where the source and target are the same, in order to cover that in the tests I need a line in the dataset to have the same source and target.

**Pull request merged**